static/js/tools/statsvar_menu.tsx

```diff
@@ -104,15 +106,21 @@ class Node extends Component<NodePropType, NodeStateType> {
     );
   };
   private _handleCheckboxClick = (): void => {
-    this.setState({
-      checked: !this.state.checked,
-    });
-    updateUrl({
-      statsVarPath: {
-        statsVar: this.props.sv + this.state.nodePath,
-        shouldAdd: !this.state.checked,
-      },
-    });
+    // Perform onClick override if provided.
+    if (this.props.onClick) {
+      this.props.onClick(this.props.sv)
```

Comment:
Can include the path info in the url too, there could be multiple nodes for the same statsvar

Reply:
Actually, we can pass updateUrl/onClick as a property function, then no need to override.